storage3/types.py

```diff
@@ -41,9 +41,11 @@ class ListBucketFilesOptions(TypedDict):
 
 
 class TransformOptions(TypedDict):
-    height: Optional[float]
-    width: Optional[float]
+    height: Optional[int]
+    width: Optional[int]
     resize: Optional[Union[Literal["cover"], Literal["contain"], Literal["fill"]]]
+    format: Optional[Union[Literal["origin"], Literal["avif"]]]
```

Comment:
```suggestion
    resize: Literal["cover", "contain", "fill"]
    format: Literal["origin", "avif"]
```
[you can pass all the literals directly to `Literal`](https://docs.python.org/3/library/typing.html#typing.Literal)